can/io/blf.py

```diff
@@ -236,6 +250,29 @@ def __iter__(self):
                                       data=can_data[:length],
                                       channel=channel - 1)
                         yield msg
+                    elif obj_type == CAN_FD_MESSAGE_64:
+                        (
+                            channel, dlc, _, _, can_id, _, fd_flags
+                         ) = CAN_FD_MSG_64_STRUCT.unpack_from(data, pos)[:7]
+                        length = dlc2len(dlc)
+                        can_data = struct.unpack_from(
+                            "<{}s".format(length),
+                            data,
+                            pos + CAN_FD_MSG_64_STRUCT.size
+                        )[0]
+                        msg = Message(
+                            timestamp=timestamp,
+                            arbitration_id=can_id & 0x1FFFFFFF,
+                            is_extended_id=bool(can_id & CAN_MSG_EXT),
+                            is_remote_frame=bool(fd_flags & REMOTE_FLAG_64),
+                            is_fd=bool(fd_flags & EDL_64),
+                            bitrate_switch=bool(fd_flags & BRS_64),
+                            error_state_indicator=bool(fd_flags & ESI_64),
+                            dlc=length,
+                            data=can_data[:length],
```

Comment:
It would probably be more efficient something like this instead of unpacking the raw data above:

```python
data_offset = pos + CAN_FD_MSG_64_STRUCT.size

data[data_offset:data_offset + length]
```

Also I've realized it should really be the valid payload length and not the DLC to support remote frames correctly, but it is more of a sanity thing.